Makefile

```diff
@@ -163,6 +163,14 @@ _opt_pylint:
 	@command -v flake8 && { $(MAKE) pylint; exit $$?; } \
 		|| echo "SKIP: pylint (flake8 not found)"
 
+current_branch=$(shell git branch --show-current)
+ancestor_commit=$(shell git merge-base master $(current_branch))
+commits_on_current_pr=$(shell git rev-list $(ancestor_commit)..$(current_branch))
+commitlint:
+	$(foreach commit, $(commits_on_current_pr), \
+	git show -s --format=%s $(commit) | nvim -Es +"%p"; \
```

Comment:
maybe use `$(realpath build/bin/nvim)` instead of whatever is in $PATH ?

Reply:
Fixed it, check if it looks alright please. Had to add `-u NONE` flags to avoid errors when starting neovim.